docker-compose.yml

```diff
@@ -215,6 +215,11 @@ services:
       - "PROXY_METHOD=${PROXY_METHOD}"
       - "PROXY_USER=${PROXY_USER}"
       - "PROXY_PASSWORD=${PROXY_PASSWORD}"
+      # s3 settings
+      - "S3_BUCKET=${S3_BUCKET}"
+      - "S3_ENDPOINT=${S3_ENDPOINT}"
+      - "S3_ACCESS_KEY=${S3_ACCESS_KEY}"
+      - "S3_SECRET_KEY=${S3_SECRET_KEY}"
```

Comment:
Can you also add the SUPERVISOR env vars, otherwise they would not get propagated